### PR TITLE
Updated the docs for Query Actor creation

### PR DESCRIPTION
**Pull request by technical-harsh90**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @technical-harsh90,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by technical-harsh90** [comment]
I have signed the CLA.

**Comment by ktoso** [comment]
OK TO TEST

**Review by johanandren — APPROVED** [review]
LGTM, I guess.

File: akka-docs/src/main/paradox/guide/tutorial_5.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ detect devices that stopped before they got the `ReadTemperature` message we wil
 way, we get `Terminated` messages for those that stop during the lifetime of the query, so we don't need to wait
 until the timeout to mark these as not available.
 
-Putting this together, the outline of our `DeviceGroupQuery` actor looks like this:
+Putting this together, the outline of our `DeviceGroupQuery` actor looks like below, however we have not defined the receive method in it:
```
Comment (patriknw) [inline akka-docs/src/main/paradox/guide/tutorial_5.md]: in it *yet*?

**Comment by chbatey** [comment]
@technical-harsh90 do you plan on finishing this one? Feel free to re-open if so!

**Pull request closed without merging**